### PR TITLE
Correct

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
This sentence would imply that by default all steps get rejected.

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
What it means to say is that `isoutofdomain(u,p,t) = false` is the default (this part needs the argument update)

File: docs/src/basics/common_solver_opts.md [hunk 1]
```diff
@@ -176,9 +176,9 @@ explanations of the timestepping algorithms, see the
 * `callback`: Specifies a callback. Defaults to a callback function which
   performs the saving routine. For more information, see the
   [Event Handling and Callback Functions manual page](../../features/callback_functions.html).
-* `isoutofdomain`: Specifies a function `isoutofdomain(t,u)` where, when it
+* `isoutofdomain`: Specifies a function `isoutofdomain(u,t)` where, when it
```
Comment (ChrisRackauckas) [inline docs/src/basics/common_solver_opts.md]: u,p,t. It's seen here: https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/common_defaults.jl

**Comment by BeastyBlacksmith** [comment]
I hope i got it all right now

**Comment by ChrisRackauckas** [comment]
Actually it looks like somewhere along the line the default for that was changed to check for NaNs.

**Comment by BeastyBlacksmith** [comment]
true, i'll change that

**Comment by BeastyBlacksmith** [comment]
no, thats `unstable_check` that does it. So everything should be fine now.

**Comment by ChrisRackauckas** [comment]
Thanks!

**Pull request merged**